README.md

```diff
@@ -92,7 +92,7 @@ Which `.vsix` file (or files) to publish. This controls what value will be used
 
 | Variable      | Description                                                                                                                                                                                                                                                                                                     |
 | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `VSCE_PAT`    | **Required** (unless `publish` is set to `false`). The personal access token to publish the extension to Visual Studio Marketplace                                                                                                                                                                              |
+| `VSCE_PAT`    | **Required** (unless `publish` is set to `false`, or the `OVSX_PAT` environment variable is set). The personal access token to publish the extension to Visual Studio Marketplace                                                                                                                               |
```

Comment:
Now VSCE_PAT is just as important (or not important) as OVSX_PAT.

We should find some way to describe them similarly.

Maybe the following is enough:

VSCE_PAT _Optional_. The personal access token to publish to Visual Studio Marketplace.

Reply:
I have updated the docs: 79c40e0fc1ded0a2d806d8d30af2b72beb61cbec.